### PR TITLE
Emogrifier update to 1.1.0

### LINKED ISSUE #6946: Outdated pelago/emogrifier package
I'm providing just one example of possible bugs here. The problem is that version of pelago/emogrifier package that Magento 2 uses was released almost 1,5 year ago.
### Preconditions
1. Magento 2.1.1
### Steps to reproduce
1. Custom email-inline styles, compressed (i.e. by sass compiler)
2. `parent > children` selector used in styles (in compressed file it looks `parent>children`)
### Expected result
1. Styles should be applied correctly (description in `_email-base.less` file says that descendant is supported.
### Actual result
1. Styles are not applied due to Emogrifier bug, that was already fixed and correct version is available.

When do you plan to update this package?

### PR DESCRIPTION
**Pull request by Bartlomiejsz**
Fixes #6946 and #4737

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "composer/composer": "1.0.0-beta1",
         "monolog/monolog": "1.16.0",
         "oyejorge/less.php": "~1.7.0",
-        "pelago/emogrifier": "0.1.1",
+        "pelago/emogrifier": "1.1.0",
```
Comment (orlangur) [inline composer.json]: Don't forget to commit changed `composer.lock` ;)

**Review by orlangur — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Bartlomiejsz) [reply composer.json]: Yes, you're right, thanks :)

**Review by orlangur — APPROVED** [review]

**Review by orlangur — APPROVED** [review]

**Comment by vrann** [comment]
@Bartlomiejsz please update the code with the latest develop branch

**Comment by Bartlomiejsz** [comment]
@vrann, @orlangur I'm closing it, I've created new PR from the develop branch: #9327

**Comment by orlangur** [comment]
@Bartlomiejsz ok, let's go this way now, but generally it's better to force push into the same branch after you rebased against latest mainline ;)

**Comment by Bartlomiejsz** [comment]
@orlangur ok, next time I will remember :)

**Pull request closed without merging**